docs/guide/breadcrumbs/breadcrumbs_api/routesbreadcrumbsgenerator.md

```diff
@@ -1,18 +1,17 @@
 # RoutesBreadcrumbsGenerator
 
-This generates the breadcrumbs for the symfony routes.
+`Silversolutions\Bundle\EshopBundle\Service\Breadcrumbs\RoutesBreadcrumbsGenerator`
+generates the breadcrumbs for the Symfony routes.
```

Comment:
Two paragraphs up the name of the method was preceded by a word "Method..." For the sake of consistency I would choose either way, my suggestion is to follow this example, meaning to make the sentence begin with the name of the method only.